### PR TITLE
all tests pass

### PR DESCRIPTION
**Pull request by pyramation**

### CODE REVIEW

File: src/lib/connection.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  noWarnings: true
+})
+
+export const connect = async (connection: any): Promise<any> => {
+  const cn = await pgp(connection)
+  const db = await cn.connect()
+  return db
+}
+
+export const close = (db: any): void => {
```
Comment (lifeiscontent) [inline src/lib/connection.ts]: @pyramation if you don't need to I'd create `db` locally in this file so you don't need to pass it around.

File: src/lib/types.ts [hunk 2]
```diff
@@ -1,7 +1,16 @@
 export interface ConnectionObject {
   database: string
   password: string
-  port: string
+  port: string | number
```
Comment (lifeiscontent) [inline src/lib/types.ts]: @pyramation I don't think these are necessary, you should use the types from the library. https://github.com/vitaly-t/pg-promise/blob/master/typescript/pg-subset.d.ts

File: test/utils.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 30000
+require('dotenv').load()
```
Comment (lifeiscontent) [inline test/utils.ts]: @pyramation you can require dotenv from the npm start command.

Comment (lifeiscontent) [reply test/utils.ts]: E.g. https://www.npmjs.com/package/dotenv

Comment (pyramation) [reply test/utils.ts]: you have an example?

File: package.json [hunk 4]
```diff
@@ -94,10 +92,12 @@
     "tslint-config-standard": "^6.0.0",
     "typedoc": "^0.9.0",
     "typescript": "^2.3.4",
-    "uuid": "^3.1.0",
     "validate-commit-msg": "^2.12.2"
   },
   "dependencies": {
-    "streamify-string": "^1.0.1"
+    "pg-promise": "^6.10.3",
```
Comment (lifeiscontent) [inline package.json]: @pyramation you might need to include your dependencies in the rollup config.

File: src/lib/connection.ts (hunk 1 above)
Comment (lifeiscontent) [reply src/lib/connection.ts]: when you instantiate variables in files and use them only within the file, they act like singletons.

File: src/lib/types.ts (hunk 2 above)
Comment (lifeiscontent) [reply src/lib/types.ts]: @pyramation I'm not sure how that simplifies the problem, interfaces only act as linters to the code. There shouldn't be a conflict during runtime/build

File: test/utils.ts (hunk 3 above)
Comment (lifeiscontent) [reply test/utils.ts]: Yes, this is the example: https://www.npmjs.com/package/dotenv#preload

File: package.json (hunk 4 above)
Comment (pyramation) [reply package.json]: that's it!

**Pull request merged**